### PR TITLE
Create; initial design for carousel

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

1. to get a carousel for mobile only

Changes proposed in this pull request:

- create a demo with both a polyfill example and non polyfill
- create helper css

### CODE REVIEW

File: examples/carousel/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                  Harvard
+                </footer>
+              </blockquote>
+              <a href="" class="btn text-center text-uppercase Stories-more">Read the Story</a>
+            </article>
+          </div>
+        </div>
+      </div>
+    </section>
+    <script src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/scroll-snap-polyfill.js"></script>
```
Comment (jgallen23) [inline examples/carousel/index.html]: lets remove this by default.

File: examples/carousel/no-fill.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```
Comment (jgallen23) [inline examples/carousel/no-fill.html]: we can kill this page

File: styles/helpers/carousel.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@media (--mobile) {
+  .row-sm-carousel {
+    @mixin row-carousel;
```
Comment (jgallen23) [inline styles/helpers/carousel.css]: missing mixin file?

Comment (chrisopedia) [inline styles/helpers/carousel.css]: yup

File: examples/carousel/no-fill.html (hunk 2 above)
Comment (jgallen23) [inline examples/carousel/no-fill.html]: yeah, the polyfill makes things worse

**Pull request merged**